spidermon/contrib/actions/sentry/sentry.py

```diff
@@ -0,0 +1,46 @@
+from spidermon.contrib.actions.sentry import SendMessage
+from spidermon.exceptions import NotConfigured
+
+from sentry_sdk.client import Client
+from sentry_sdk import configure_scope
+
+
+class SendSentryMessage(SendMessage):
+    def __init__(self, sentry_log_level=None, sentry_dsn=None, sentry_enabled=None, *args, **kwargs):
+        super(SendSentryMessage, self).__init__(*args, **kwargs)
+
+        self.sentry_log_level = sentry_log_level or self.sentry_log_level
+        self.sentry_dsn = sentry_dsn or self.sentry_dsn
+        self.sentry_enabled = sentry_enabled or self.sentry_enabled
+
+    @classmethod
+    def from_crawler_kwargs(cls, crawler):
+        kwargs = super(SendSentryMessage, cls).from_crawler_kwargs(crawler)
+        kwargs.update({
+            'sentry_dsn': crawler.settings.get('SENTRY_DSN'),
+            'sentry_enabled': crawler.settings.get('SENTRY_ENABLED'),
+            'sentry_log_level': crawler.settings.get('SENTRY_LOG_LEVEL'),
+        })
+        return kwargs
+
+    def send_message(self, message):
+        if not self.sentry_enabled:
+            raise NotConfigured("SENTRY_ENABLED Should be turn on")
+
+        if not self.sentry_dsn:
+            raise NotConfigured("Missing SENTRY_DSN setting")
+
+        sentry_client = Client(self.sentry_dsn)
+
+        with configure_scope() as scope:
+            scope.set_extra("job_link", message.get('job_link', ''))
+            scope.set_extra("spider_name", message.get('spider_name', ''))
+            scope.set_extra("spider_stats", message.get('spider_stats', {}))
+            scope.set_extra("items_count", message.get('items_count', 0))
+
+            scope.set_extra("passed_monitors_count", message.get('passed_monitors_count', 0))
+            scope.set_extra("failed_monitors_count", message.get('failed_monitors_count', 0))
+
+            scope.set_extra("failed_monitors", message.get('failed_monitors', []))
+
+            sentry_client.capture_event({"message": message.get('title'), 'level': 'error'}, scope=scope)
```

Comment:
Why did you fixed the level of the message as an error? What if I want to register the result of my spider even if there is no errors?

Reply:
Going to pass the value of SPIDERMON_SENTRY_LOG_LEVEL